### PR TITLE
Cleanup Spark Program Logs

### PR DESCRIPTION
**Pull request by deepakwadhwani**
- Move debugs to debug level.
- Add missing Spark configurations that were causing WARN logs.
- Move chatty loggers to WARN.

### CODE REVIEW

File: cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml [hunk 1]
```diff
@@ -36,6 +36,9 @@
   <logger name="SecurityLogger.org.apache.hadoop.ipc.Server" level="WARN"/>
   <!-- Because of HADOOP-11180 move this to Error -->
   <logger name="org.apache.hadoop.security.token.Token" level="ERROR"/>
+  <logger name="akka.event.slf4j.Slf4jLogger" level="WARN"/>
```
Comment (chtyim) [inline cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml]: You probably can have all `akka` set at `WARN` level.

Comment (deepakwadhwani) [reply cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml]: Those are the two I saw in the logs. Are there other loggers that I should add?

Comment (chtyim) [reply cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml]: In general, logs from akka are useless...

File: cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/distributed/SparkExecutionService.java [hunk 2]
```diff
@@ -85,7 +85,7 @@
   public SparkExecutionService(LocationFactory locationFactory, String host,
                                ProgramRunId programRunId, @Nullable WorkflowToken workflowToken) {
     this.locationFactory = locationFactory;
-    this.httpServer = NettyHttpService.builder(SparkExecutionService.class.getName())
+    this.httpServer = NettyHttpService.builder(programRunId.getProgram())
```
Comment (chtyim) [inline cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/distributed/SparkExecutionService.java]: Don't just use the program name. It is the name for the http server, so better be `programRunId.getProgram() + "-spark-exec-service"`. The name is not just for logging, but also used to name threads.

**Review by chtyim — APPROVED** [review]
LGTM, please squash before merging

**Pull request merged**